tests/integration/it_auth_test.py

```diff
@@ -28,25 +30,22 @@ def test_incorrect_password(db_session):
     db_session.commit()
     assert (login("unknown", "unknown") == "login.failed")
 
-"""
-@todo #149:30min Continue implementation of the login function. The
- login("vgv", "pass") is returning ValueError: not a valid bcrypt_sha256 hash.
- Information from https://passlib.readthedocs.io/en/stable/index.html can be
- used to help.
-"""
+
 def test_login(db_session):
     employee = Employee(first_name="Alice", last_name="Cooper",
                         username="vgv", phone_number="1", account_status="A",
                         birth_date=datetime.utcnow(), pin_code=4567,
                         registration_date=datetime.utcnow(), user_status="U",
-                        email="test@test.com", password="pass")
+                        email="test@test.com", password=auth_hash("pass"))
     db_session.add(employee)
     db_session.commit()
-    """error = login("vgv", "pass")"""
-    error = login("unknown", "unknown");
+    error = login("vgv", "pass")
+    assert not error
+    assert flask.session['user_id'] == employee.id
     db_session.delete(employee)
     db_session.commit()
     db_session.remove()
+    error = login("unknown", "unknown");
```

Comment:
@ekondrashev do you need to remove the employee before testing `unknown` login?

Reply:
@ammaratef45 that is what test was doing, I believe it is a part of a cleanup, so I'd move this to try/finally, but yes, the deletion is necessary imo

Reply:
@ekondrashev there is a fixture that empty the DB I think, need to revise that
But at least make it in a final block or a fixture like you said